packages/graphql-relational-schema-transformer/src/RelationalDBTemplateGenerator.ts

```diff
@@ -0,0 +1,199 @@
+     * 
+     * @returns the created CloudFormation template.
+     */
+    public createTemplate(context: any): Template {
+        const template =  {
+            AWSTemplateFormatVersion: "2010-09-09",
+            Parameters: this.makeParameters(this.context.databaseName),
+            Resources: {
+                [ResourceConstants.RESOURCES.RelationalDatabaseDataSource]: this.makeRelationalDataSource(context),
+                [ResourceConstants.RESOURCES.RelationalDatabaseAccessRole]: this.makeIAMDataSourceRole()
+            }
+        }
+
+        return template
+    }
+
+    /**
+     * Provided a Cloudform Template, this method adds Resolver Resources to the
+     * Template.
+     *
+     * @param template - the Cloudform template
+     * @returns the given template, updated with new resolvers.
+     */
+    public addRelationalResolvers(template: Template) : Template {
+        let resolverGenerator = new RelationalDBResolverGenerator(this.context)
+        template.Resources = {...template.Resources, ...resolverGenerator.createRelationalResolvers()}
+        return template
+    }
+
+    /**
+     * Provided a Cloudform Template, this method returns the cfn json template as a string
+     *
+     * @param template - the Cloudform template
+     * @returns the json, string form of the template given.
+     */
+    public printCloudformationTemplate(template: Template): string {
+        return cloudform(template)
+    }
+
+    /*
+     * Private Helper Methods for Generating the Necessary CFN Specs for the CFN Template
+     */
+
+    /**
+     * Creates any Parmaters needed for the CFN Template
+     * 
+     * @param databaseName - the name of the database being parsed.
+     * @returns the parameters for the template.
+     */
+    private makeParameters(databaseName: string) {
+        return {
+            [ResourceConstants.PARAMETERS.AppSyncApiName]: new StringParameter({
+                Description: `The name of the AppSync API generated from database ${databaseName}`,
+                Default: `AppSyncSimpleTransform`
+            }),
+            [ResourceConstants.PARAMETERS.Env]: new StringParameter({
+                Description: 'The environment name. e.g. Dev, Test, or Production',
+                Default: 'NONE'
+            }),
+            [ResourceConstants.PARAMETERS.S3DeploymentBucket]: new StringParameter({
+                Description: 'The S3 bucket containing all deployment assets for the project.'
+            }),
+            [ResourceConstants.PARAMETERS.S3DeploymentRootKey]: new StringParameter({
+                Description: 'An S3 key relative to the S3DeploymentBucket that points to the root of the deployment directory.'
+            }),
+            [ResourceConstants.PARAMETERS.AppSyncApiId]: new StringParameter({
+                Description: 'The id of the AppSync API associated with this project.'
+            }),
+            [ResourceConstants.PARAMETERS.rdsRegion]: new StringParameter({
```

Comment:
Can we move the rds constants to this package? This class isn't loaded by the transformer libs so I think the extra params should live outside of it.

Reply:
done